.github/actions/test-type-vars/action.yml

```diff
@@ -71,6 +71,13 @@ runs:
         echo "TEST_CHART_DIR=${charts_base_dir}/${{ inputs.chart-dir }}" | tee -a $GITHUB_ENV
         echo "ABSOLUTE_TEST_CHART_DIR=${GITHUB_WORKSPACE}/${CHART_PATH}" | tee -a $GITHUB_ENV
 
+        # configuring TEST_HELM_EXTRA_ARGS
         if [[ "${{ inputs.enterprise }}" == "true" ]]; then
-          echo "TEST_HELM_EXTRA_ARGS=--values ${charts_base_dir}/${{ inputs.chart-dir }}/values-enterprise.yaml" | tee -a $GITHUB_ENV
+          helm_extra_args="--values ${charts_base_dir}/${CHART_DIR}/values-enterprise.yaml"
         fi
+
+        if [[ -f "${CHART_PATH}/values-digest.yaml" ]]; then
```

Comment:
Given that it uses an explicit approach (unlike `values-enterprise.yaml`), we either add an input for the `digest` (to work like the `enterprise`, which is better), or we make that visible somehow (document it or so).

Reply:
I added the input